src/snowflake/snowpark/modin/plugin/extensions/base_overrides.py

```diff
@@ -861,7 +861,7 @@ def _dropna(
         raise ValueError("invalid how option: %s" % how)
     if subset is not None:
         if axis == 1:
-            indices = self.index.get_indexer_for(subset)
+            indices = self.index.to_pandas().get_indexer_for(subset)
```

Comment:
let's simply skip the check for axis = 1, that is not supported by snowpark pandas

Reply:
done!